src/main/java/com/coveros/selenified/utilities/Listener.java

```diff
@@ -107,19 +105,6 @@ private static String getTestName(ITestResult result) {
     @Override
     public void onTestStart(ITestResult result) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1185)